go/mars/go.mod

```diff
@@ -0,0 +1,5 @@
+module main
+
+go 1.12
```

Comment:
could we remove this line ? 
+ I know go generate it, but we will not stick to a particular go version
+ We are already in `go` **1.13**

Reply:
Actually that line specifies the minimal version of Go that this code will compile with. I removed it anyway.